src/app/containers/ChartbeatAnalytics/index.test.jsx

```diff
@@ -120,35 +118,31 @@ describe('Charbeats Analytics Container', () => {
       {},
     );
     expect(testUtils.getConfig).toHaveBeenCalledTimes(1);
-    expect(tree).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   it('should return null when toggle is disbaled for live', () => {
-    const tree = renderer
-      .create(
-        <ContextWrap platform="canonical" pageType="article" origin="bbc.com">
-          <ChartbeatAnalytics data={frontPageData} />
-        </ContextWrap>,
-      )
-      .toJSON();
-
-    expect(tree).toMatchSnapshot();
+    const { container } = render(
+      <ContextWrap platform="canonical" pageType="article" origin="bbc.com">
+        <ChartbeatAnalytics data={frontPageData} />
+      </ContextWrap>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
based on the title of this test I would change the assertion to `expect(container.firstChild).toBeNull();`

Reply:
Fixed!